src/app/shared/models/user.model.ts

```diff
@@ -1,5 +1,6 @@
 export interface User {
   id: string,
+  roles: Array<string>,
```

Comment:
in typescript usually you jsut define arrays like this:
string[]

so type + []